tests/foreman/ui/test_ansible.py

```diff
@@ -742,9 +744,37 @@ def test_positive_install_ansible_collection_via_job_invocation(self):
             6. Click "Submit"
 
         :expectedresults: The Ansible collection is successfully installed on the host
-
-        :CaseAutomation: NotAutomated
         """
+        client = rhel_contenthost
+        # Enable Ansible repository and Install ansible or ansible-core package
+        client.register(module_org, None, module_ak_with_cv.name, target_sat)
```

Comment:
can we use `api_register` here instead of cli method?

Reply:
I don't think, there will be any difference if we use API instead of CLI to do the setup, so we can skip this